packages/lib/src/components/Giftcard/Giftcard.tsx

```diff
@@ -74,6 +74,15 @@ export class GiftcardElement extends UIElement {
         }
     };
 
+    protected handleOrder = (order: Order) => {
+        this.elementRef._parentInstance.update({ order });
+        if (this.props.session && this.props.onOrderCreated) {
+            return new Promise((resolve, reject) => {
```

Comment:
Does this _need_ to return a Promise? 
The function it overrides `UIElement.handleOrder` doesn't.

Reply:
It has to do with keeping the same API has we have for other session callback like `onOrderRequest` and `onBalanceCheck`.

Reply:
ok